packages/turf-convex/test.ts

```diff
@@ -32,3 +33,17 @@ test("turf-convex -- empty", (t) => {
   t.deepEqual(convex(featureCollection([])), null, "corner case: null hull");
   t.end();
 });
+
+test("turf-convex -- properties are returned", (t) => {
```

Comment:
Could you please make the test name more descriptive? Maybe "properties are transferred to result polygon" instead?

Reply:
Sure. Thanks for the feedback. I'll get back to this as soon as back from holidays.